pkg/commands/drift/command.go

```diff
@@ -85,6 +90,46 @@ func (c *DetectIamDriftCommand) Flags() *cli.FlagSet {
 		Usage:   `The maximum number of concurrent requests allowed at any time to GCP.`,
 		Default: 10,
 	})
+	f.BoolVar(&cli.BoolVar{
+		Name:    "skip-github-issue",
+		Target:  &c.flagSkipGitHubIssue,
+		Example: "true",
+		Usage:   `Whether or not to create a GitHub Issue when a drift is detected.`,
+		Default: false,
+	})
+	f.StringVar(&cli.StringVar{
+		Name:    "github-token",
+		Target:  &c.flagGitHubToken,
+		Usage:   `The github token to use to authenticate to create & manage GitHub Issues.`,
+		Default: "",
+	})
+	f.StringVar(&cli.StringVar{
+		Name:    "github-owner",
+		Target:  &c.flagGitHubOwner,
+		Usage:   `The github token to use to authenticate to create & manage GitHub Issues.`,
+		Default: os.Getenv("GITHUB_REPOSITORY_OWNER"),
+	})
+	f.StringVar(&cli.StringVar{
+		Name:    "github-repo",
+		Target:  &c.flagGitHubRepo,
+		Example: "guardian",
+		Usage:   `The github token to use to authenticate to create & manage GitHub Issues.`,
+		Default: os.Getenv("GITHUB_REPOSITORY"),
```

Comment:
This envvar is the full repo name: owner/repo not just the name.

Reply:
Discussed offline. Removed env var for time being.